### PR TITLE
Aressem/add gtest

### PR DESCRIPTION
**Pull request by aressem**

### CODE REVIEW

File: bootstrap-cmake.sh [hunk 1]
```diff
@@ -23,8 +23,8 @@ fi
 cmake3 \
     -DCMAKE_INSTALL_PREFIX=/opt/vespa \
     -DJAVA_HOME=/usr/lib/jvm/java-openjdk \
-    -DEXTRA_LINK_DIRECTORY="/opt/vespa-boost/lib;/opt/vespa-cppunit/lib;/usr/lib64/llvm3.9/lib" \
-    -DEXTRA_INCLUDE_DIRECTORY="/opt/vespa-boost/include;/opt/vespa-cppunit/include;/usr/include/llvm3.9" \
-    -DCMAKE_INSTALL_RPATH="/opt/vespa/lib64;/opt/vespa-boost/lib;/opt/vespa-cppunit/lib;/usr/lib/jvm/java-1.8.0/jre/lib/amd64/server;/usr/lib64/llvm3.9/lib" \
+    -DEXTRA_LINK_DIRECTORY="/opt/vespa-gtest/lib;/opt/vespa-boost/lib;/opt/vespa-cppunit/lib;/usr/lib64/llvm3.9/lib" \
+    -DEXTRA_INCLUDE_DIRECTORY="/opt/vespa-gtest/lib;/opt/vespa-boost/include;/opt/vespa-cppunit/include;/usr/include/llvm3.9" \
```
Comment (vekterli) [inline bootstrap-cmake.sh]: Should this be `/opt/vespa-gtest/include` and not `lib`?

**Review by vekterli — COMMENTED** [review]
See include path question

File: bootstrap-cmake.sh (hunk 1 above)
Comment (aressem) [reply bootstrap-cmake.sh]: Good catch. Fixed in new commit.

**Review by vekterli — APPROVED** [review]
👌

**Pull request merged**